Add missing volume streams to sensor and command

dshokouhi opened the pull request:
<!-- Thank you for submitting a Pull Request and helping to improve Home Assistant. Please complete the following sections to help the processing and review of your changes. Please do not delete anything from this template. -->

## Summary
<!-- Provide a brief summary of the changes you have made and most importantly what they aim to achieve -->

Noticed that we were missing some of the volume streams, not all devices show to the user all the streams so better to cover all our bases. Samsung devices for example show System volume while others do not. All are still controllable and report data.

## Screenshots
<!-- If this is a user-facing change not in the frontend, please include screenshots in light and dark mode. -->

## Link to pull request in Documentation repository
<!-- Pull requests that add, change or remove functionality must have a corresponding pull request in the Companion App Documentation repository (https://github.com/home-assistant/companion.home-assistant). Please add the number of this pull request after the "#" -->
Documentation: home-assistant/companion.home-assistant#720

## Any other notes
<!-- If there is any other information of note, like if this Pull Request is part of a bigger change, please include it here. -->

jpelgrom commented:
If the point is to support all possible audio streams, shouldn't there also be support for `STREAM_ACCESSIBILITY` and `STREAM_DTMF`?

dshokouhi commented:
Looks like we are not allowed to change the volume of accessibility streams 🙃 

![image](https://user-images.githubusercontent.com/1634145/159774029-6ece0fc9-896a-4390-96c4-3f9f82ab02ff.png)

dshokouhi commented:
Ok added in Accessibility and DTMF volume sensors, also added 1 missing audio mode (the other 2 we can add once we target the next version of android). Also added volume level control for DTMF.
